pkg/transaction/transaction.go

```diff
@@ -403,14 +417,19 @@ func (t *transactionService) ResendTransaction(ctx context.Context, txHash commo
 		return err
 	}
 
+	gasFeeCap, gasTipCap, err := t.suggestedFeeAndTip(ctx, sctx.GetGasPrice(ctx), storedTransaction.GasTipBoost)
+	if err != nil {
+		return err
+	}
+
 	tx := types.NewTx(&types.DynamicFeeTx{
 		Nonce:     storedTransaction.Nonce,
 		ChainID:   t.chainID,
 		To:        storedTransaction.To,
 		Value:     storedTransaction.Value,
 		Gas:       storedTransaction.GasLimit,
-		GasTipCap: storedTransaction.GasPrice,
-		GasFeeCap: storedTransaction.GasPrice,
+		GasTipCap: gasTipCap,
+		GasFeeCap: gasFeeCap,
```

Comment:
Definitely seems to make more sense to use updated numbers if we're resending the transaction.   But @mfw78 told me a while back that any change like this will also change the resulting transaction ID which might mess up the transaction watcher?

Reply:
I was wanting the resend to boost the price when he described the extended ramifications of doing such a thing to an existing transaction nonce.  But if the watcher is only watching for the nonce and not the specific transaction ID, then I guess this would work?